src/core_modules/capture-core/components/FormFields/New/HOC/messages/withSearchHelpMessage.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import i18n from '@dhis2/d2-i18n';
+import { colors } from '@dhis2/ui';
+import { withStyles, WithStyles } from 'capture-core-utils/styles';
+import { searchOperators } from '../../../../../metaDataMemoryStoreBuilders';
+
+const styles = (theme: any) => ({
+    help: {
+        marginLeft: 0,
+        paddingLeft: 0,
+        color: colors.grey700,
```

Comment:
To allign more with the design, I suggest:
```suggestion
        color: colors.grey600,
        marginTop: 4,
```